docs/userguide/running-unit-tests.md

```diff
@@ -120,14 +120,6 @@ end;
 ```
 Executes all tests from package _hr.test_apply_bonus_ and all tests from schema _cust_.
 
-```sql linenums="1"
```

Comment:
Instead of deleting this example, a better description of difference between this and example above is needed.